include/globals.h

```diff
@@ -119,7 +120,7 @@
 //
 // BUGBUG (davepl): If you know a cleaner way, please improve this!
 
-#define FLASH_VERSION         36    // Update ONLY this to increment the version number
+#define FLASH_VERSION          37    // Update ONLY this to increment the version number
```

Comment:
I've also fixed some wifi server issues a few checking ago that I want to pick up for my home use, so need to bump the version so I can telll one from the other when looking at the screen!

Reply:
Well, it's been a while. Maybe a version bump was overdue. :)